migrations/10.0.md

```diff
@@ -0,0 +1,42 @@
+# Major 10.0 Migration guide
+> [!IMPORTANT]
+> Please make sure to first follow the ["Upgrade new version guide"](https://github.com/AmadeusITGroup/otter/blob/release/main/docs/core/UPGRADE_NEW_VERSION.md) before going through these steps.
+
+
+## Personalization
+Customization now only relies on the ```@O3rComponent``` decorator.<br>
+The following interfaces and decorator (already deprecated) have been fully removed.
+* ``@InputMerge`` decorator
+* ``Blocks``, ``Page``, ``ExposedComponent`` interfaces
+
+These change have already been automated in the v9 ng updates.
+
+## utils.Date
+* ``dateInstance`` has been removed as it was only returning the instance of the object
+* ``getUtilsDate`` used to create a ``utils.Date`` instance from a ``utils.DateTime`` instance has been removed.<br>
+Please use instead:
+```typescript
+const myUtilsDateTimeInstance = new utils.DateTime();
+const myDate = new utils.Date(myUtilsDateTimeInstance)
+```
+
+## Element fixtures
+The signature of ``throwOnUndefined`` has been updated and now returns a Promise.
+
+## End of @ama-sdk/generator-sdk
+``@ama-sdk/generator-sdk``, based on yeoman and the Swagger CodeGen and deprecated since Otter v8, is now completely gone and replaced by ``@ama-sdk/schematics`` running with Angular schematics and the OpenApiTools generator.
```

Comment:
```suggestion
``@ama-sdk/generator-sdk``, based on yeoman and the Swagger CodeGen is deprecated since Otter v8, is now completely gone and replaced by ``@ama-sdk/schematics`` running with Angular schematics and the OpenApiTools generator.
```